### DISCUSSION TITLE
Booleans as custom attributes of types and properties

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#844] [Q&A] [answered]
Dear Developers,

The [ALTER PROPERTY](https://docs.arcadedb.com/#SQL-Alter-Property) statement is currently documented as allowing strings and integers as attribute value types. I tested boolean values which seem to work, but this is not documented. Thus I would like to ask if this accidental, undocumented or actually not-working?

Thanks

PS: I can update the docu if Booleans are allowed but undocumented

**Comment by lvca** [comment, marked as answer]
It's correct, boolean is allowed. Actually, everything can be serialized as a JSON should work. If you can help with the docs, that would be fantastic, thanks in advance!
